src/librustc_tsan/lib.rs

```diff
@@ -18,3 +18,10 @@
             issue = "0")]
 
 extern crate alloc_system;
+
+#[cfg(not(stage0))]
+use alloc_system::System;
+
+#[cfg(not(stage0))]
+#[global_allocator]
+static ALLOC: System = System;
```

Comment:
So what would happen if this crate were imported into some other crate? In that case, does it (implicitly) get the global-allocator defined here?